src/Symfony/Component/HttpKernel/DataCollector/RequestDataCollector.php

```diff
@@ -63,6 +64,10 @@ public function collect(Request $request, Response $response, \Exception $except
             if ('_route' === $key) {
                 $route = is_object($value) ? $value->getPath() : $value;
             }
+
+            if ('_route_params' === $key && is_array($value)) {
+                $routeParams = $value;
```

Comment:
Well.. that's what broke it before :)) The value is passed to `table.html.twig` which uses `profiler_dump` already.

We could wrap the array in a `ParameterBag` (seems consistent..) and use `bag.html.twig`, but it's the same result and probably a BC break for `getRouteParams`.